### ISSUE TITLE
[Ulduar] IC Stormcaller Brundir lightning whirl

### ISSUE THREAD

**Issue by T0mken** [Stormforge-gg/Bugtracker-Fordring#1849]
Can't interrupt Stormcaller Brundir's lightning whirl when it's only him left, luckily it does no dmg when it's not interruptable.

**Comment by Aithne99** [comment]
I could kick it just fine, and it always dealt damage when left to cast. Got a video or any other manner of info on reproducing?

**Comment by Weuyn** [comment]
He had a buff on him in our run when it was just him left. Made him unkickable and deal damage when you attacked him. Lightning shield or something?

Barely managed to survive with rotating DSac/tranquility/hymn. Honestly probably harder than steelbreaker lol. On the healers, anyway.

He lost the buff for a little bit after he did his raising into the air ability, so we could kick two whirls but thats it.

**Comment by Godmatik** [comment]
The buff he gains when last one alive should only make him stun immune, not kick immune

**Comment by Weuyn** [comment]
Yeah this is weird. I was in a pug with Kanbuk on 25m, they tanked him with a paladin and sent kicker dps over to him on P2 onwards, they had no problems in phase 3. On my guild run, we have a DK tank interrupting and Brundir becomes kick immune until his first lightning tendrils phase, then we can kick him after.

Is there DR on interrupts? I'll ask the DK if they are mind freezing chain lightning.

**Comment by Iluvu-SF** [comment]
This is an ongoing issue today. If you wish I can acquire multiple testimonies from multiple guilds. I can possibly obtain video as well upon request
